### PR TITLE
Feat/memoize logs db

### PR DESCRIPTION
**Pull request by janl**

### CODE REVIEW

File: lib/dbs.js [hunk 1]
```diff
@@ -49,11 +49,25 @@ const retryHandler = {
...
+
+let currentLogsDbName = getLogsDbName()
+let currentLogsDb
+
+function getLogsDb () {
+  const logsDbName = getLogsDbName()
+  if (!currentLogsDb || logsDbName !== currentLogsDbName) {
+    // either we are just starting up and there is no db
+    // instance yet, or we switched over in the month, so
+    // we need a new db instance
```
Comment (Realtin) [inline lib/dbs.js]: will the old DB instance continue to exist forever?

Comment (janl) [reply lib/dbs.js]: it should garbage collect after this last reference is overwritten.

Comment (janl) [reply lib/dbs.js]: if not, we leak one instance per month, that’s a lot better than having ~1k+ instances on load spikes :D

Comment (janl) [reply lib/dbs.js]: good question tho!

**Pull request merged**